components/atoms/ProjectInfo.js

```diff
@@ -0,0 +1,60 @@
+import PropTypes from "prop-types";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCircleInfo } from "@fortawesome/free-solid-svg-icons";
+import { useTranslation } from "next-i18next";
+import { useState } from "react";
+
+export function ProjectInfo(props) {
+  const { t } = useTranslation("common");
+  const [showInfo, setShowInfo] = useState(false);
+  return (
+    <>
+      <div className="p-4 grid grid-cols-4 gap-x-6">
+        <strong className="col-span-1">{t("started")}</strong>
+        <p className="col-span-3">{props.dateStarted.substring(0, 10)}</p>
+        <strong className="col-span-1">{t("ended")}</strong>
+        <p className="col-span-3">{props.dateEnded.substring(0, 10)}</p>
+        <strong className="col-span-1">{t("stage")}</strong>
+        <div className="info col-span-3">
+          <p className="shrink-0">
+            {props.stage}
+            <FontAwesomeIcon
```

Comment:
This should be focusable, have focus (and maybe hover) styling. Will also either need an aria label or to announce the information hidden within for screen readers.

Reply:
Maybe chat with Celeste on what kind of functionality she wants when a user interacts with the info button

Reply:
> Maybe chat with Celeste on what kind of functionality she wants when a user interacts with the info button

Chatted with Celeste, she's ok with click state instead of hover since hover doesn't work on mobile. I wrapped the icon in a button, so it's focusable, and added aria-expanded attribute for screen reader. Also added FocusTrap as wrapper for the ProjectInfo component, so the focus is trapped when it's visible.